fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -2733,12 +2733,12 @@ public void dropDb(DropDbStmt stmt) throws DdlException {
             fullNameToDb.remove(db.getFullName());
             final Cluster cluster = nameToCluster.get(db.getClusterName());
             cluster.removeDb(dbName, db.getId());
-            editLog.logDropDb(dbName, !stmt.isNeedCheckCommittedTxns());
+            editLog.logDropDb(dbName, stmt.isForceDrop());
```

Comment:
I suggest to modify it like this:
```
DropDbInfo info = new DropDbInfo(dbName, stmt.isForceDrop());
editLog.logDropDb(info);
```

Just like other `editLog.logxxx()`

Reply:
fix done